fili-wikipedia-example/README.md

```diff
@@ -65,6 +65,51 @@ Here are some sample queries that you can run to verify your server:
 
       GET http://localhost:9998/v1/data/wikipedia/hour/?format=debug&metrics=count&dateTime=PT72H/current
 
+## Running Fili with Docker
+
+There is a [Docker image](https://hub.docker.com/r/mpardesh/fili/) for Fili which can be found on 
+Dockerhub. If you would like to experiment with Fili without having to download its dependencies, you can 
+[install](https://www.docker.com/community-edition) and start Docker. Then run these commands: 
+
+    
+    docker pull mpardesh/fili 
+    docker run --name fili-wikipedia-example -i --rm -p 3001:8081 -p 3000:8082 mpardesh/fili:1.0
```

Comment:
In response to your [earlier comment](https://github.com/yahoo/fili/pull/429#discussion_r128089426) it looks like you must have already had something using that port.

The first part of the port argument is the host port, so you could always do something like `-p 9996:9998`, and then check the [here](http://localhost:9996/v1/tables) to make sure it connected, but it should forward some port to the docker container.